bcipy/signal/model/evaluate/fusion.py

```diff
@@ -0,0 +1,370 @@
+    Args:
+        eeg_data: Raw EEG data. Test data will be extracted and selected along with gaze data.
+        gaze_data: Raw Gaze data.
+        device_spec_eeg: Device specification for EEG data.
+        device_spec_gaze: Device specification for Gaze data.
+        symbol_set: Set of symbols used in the experiment. (Default = alphabet())
+        parameters: Parameters file containing the experiment-specific parameters.
+        data_folder: Folder containing the raw data and the results.
+    Returns:
+        eeg_acc: accuracy of the EEG model only
+        gaze_acc: accuracy of the gaze model only
+        fusion_acc: accuracy of the fusion
+    """
+# Extract relevant session information from parameters file
+    trial_window = parameters.get("trial_window")
+    if trial_window is None:
+        trial_window = (0.0, 0.5)
+    window_length = trial_window[1] - trial_window[0]  # eeg window length, in seconds
+
+    prestim_length = parameters.get("prestim_length")
+    trials_per_inquiry = parameters.get("stim_length")
+    # The task buffer length defines the min time between two inquiries
+    # We use half of that time here to buffer during transforms
+    buffer = int(parameters.get("task_buffer_length") / 2)
+
+    # Get signal filtering information
+    transform_params = parameters.instantiate(ERPTransformParams)
+    downsample_rate = transform_params.down_sampling_rate
+    # static_offset = parameters.get("static_trigger_offset")
+    static_offset = 0.0
+    # Get the flash time (for gaze analysis)
+    flash_time = parameters.get("time_flash")
+
+    eeg_channels = eeg_data.channels
+    eeg_type_amp = eeg_data.daq_type
+    eeg_sample_rate = eeg_data.sample_rate
+
+    gaze_channels = gaze_data.channels
+    gaze_type_amp = gaze_data.daq_type
+    gaze_sample_rate = gaze_data.sample_rate
+
+    # setup filtering
+    default_transform = get_default_transform(
+        sample_rate_hz=eeg_sample_rate,
+        notch_freq_hz=transform_params.notch_filter_frequency,
+        bandpass_low=transform_params.filter_low,
+        bandpass_high=transform_params.filter_high,
+        bandpass_order=transform_params.filter_order,
+        downsample_factor=transform_params.down_sampling_rate,
+    )
+
+    eeg_channel_map = analysis_channels(eeg_channels, device_spec_eeg)
+    gaze_channel_map = analysis_channels(gaze_channels, device_spec_gaze)
+
+    # Channel map can be checked from raw_data.csv file or the devices.json located in the acquisition module
+    # The timestamp column [0] is already excluded.
+    eeg_channels_used = [eeg_channels[i] for i, keep in enumerate(eeg_channel_map) if keep == 1]
+
+    gaze_channels_used = [gaze_channels[i] for i, keep in enumerate(gaze_channel_map) if keep == 1]
+
+    # Define the model object before reshaping the data
+    k_folds = parameters.get("k_folds")
+    eeg_model = PcaRdaKdeModel(k_folds=k_folds)
```

Comment:
I didn't specify a default model arg for the fusion function because we haven't defined a model to save yet at this point in the offline analysis (and I don't save the models trained in `fusion.py` in any case). Therefore I'm a bit confused which way it would make sense the most